tests/test_orm.py

```diff
@@ -52,6 +52,7 @@ class Contact(Model):
         is_registered = f.CheckboxField("Registered")
         link = f.LinkField("Link", Address, lazy=True)
         birthday = f.DateField("Birthday")
+        photo = f.Field("Photo", read_only=True)
```

Comment:
Why is this just a "Field" and no "TextField"?

Reply:
It's simulating a generic Field, not a TextField. Does that result in an unreliable test?